tsconfig.base.json

```diff
@@ -10,89 +10,49 @@
       ],
-      "@sagebionetworks/openchallenges/config": [
-        "libs/openchallenges/config/src/index.ts"
-      ],
-      "@sagebionetworks/openchallenges/home": [
-        "libs/openchallenges/home/src/index.ts"
-      ],
-      "@sagebionetworks/openchallenges/login": [
-        "libs/openchallenges/login/src/index.ts"
-      ],
-      "@sagebionetworks/openchallenges/not-found": [
-        "libs/openchallenges/not-found/src/index.ts"
-      ],
+      "@sagebionetworks/openchallenges/config": ["libs/openchallenges/config/src/index.ts"],
+      "@sagebionetworks/openchallenges/home": ["libs/openchallenges/home/src/index.ts"],
+      "@sagebionetworks/openchallenges/login": ["libs/openchallenges/login/src/index.ts"],
+      "@sagebionetworks/openchallenges/not-found": ["libs/openchallenges/not-found/src/index.ts"],
       "@sagebionetworks/openchallenges/org-profile": [
         "libs/openchallenges/org-profile/src/index.ts"
       ],
-      "@sagebionetworks/openchallenges/org-search": [
-        "libs/openchallenges/org-search/src/index.ts"
-      ],
-      "@sagebionetworks/openchallenges/pages": [
-        "libs/openchallenges/pages/src/index.ts"
-      ],
-      "@sagebionetworks/openchallenges/signup": [
-        "libs/openchallenges/signup/src/index.ts"
-      ],
-      "@sagebionetworks/openchallenges/styles": [
-        "libs/openchallenges/styles/src/index.ts"
-      ],
-      "@sagebionetworks/openchallenges/team": [
-        "libs/openchallenges/team/src/index.ts"
-      ],
-      "@sagebionetworks/openchallenges/themes": [
-        "libs/openchallenges/themes/src/index.ts"
-      ],
-      "@sagebionetworks/openchallenges/ui": [
-        "libs/openchallenges/ui/src/index.ts"
-      ],
+      "@sagebionetworks/openchallenges/org-search": ["libs/openchallenges/org-search/src/index.ts"],
+      "@sagebionetworks/openchallenges/pages": ["libs/openchallenges/pages/src/index.ts"],
+      "@sagebionetworks/openchallenges/signup": ["libs/openchallenges/signup/src/index.ts"],
+      "@sagebionetworks/openchallenges/styles": ["libs/openchallenges/styles/src/index.ts"],
+      "@sagebionetworks/openchallenges/team": ["libs/openchallenges/team/src/index.ts"],
+      "@sagebionetworks/openchallenges/themes": ["libs/openchallenges/themes/src/index.ts"],
+      "@sagebionetworks/openchallenges/ui": ["libs/openchallenges/ui/src/index.ts"],
       "@sagebionetworks/openchallenges/user-profile": [
         "libs/openchallenges/user-profile/src/index.ts"
       ],
-      "@sagebionetworks/openchallenges/util": [
-        "libs/openchallenges/util/src/index.ts"
-      ],
-      "@sagebionetworks/shared/util": [
-        "libs/shared/typescript/util/src/index.ts"
-      ],
+      "@sagebionetworks/openchallenges/util": ["libs/openchallenges/util/src/index.ts"],
+      "@sagebionetworks/shared/charts": ["libs/shared/typescript/charts/src/index.ts"],
+      "@sagebionetworks/shared/charts-ngx": ["libs/shared/typescript/charts-ngx/src/index.ts"],
```

Comment:
I added paths for the two new libraries. The other changes are just related to auto-formatting.